backend/src/routes/api/status/statusUtils.ts

```diff
@@ -20,6 +20,7 @@ export const status = async (
   const { server } = currentCluster;
 
   const userName = await getUserName(fastify, request);
+  const userID: string = await getUserId(fastify, request);
```

Comment:
both `getUserName` and `getUserId` perform a network request to obtain user data. These should be combined into a single `getUserInfo` and then deconstruct the specifics you want here:
```suggestion
  const { userName, userID } = await getUserInfo(fastify, request);
```